packages/manager/apps/hub/src/dashboard/translations/Messages_fr_FR.json

```diff
@@ -1,5 +1,6 @@
 {
   "manager_hub_dashboard_welcome": "Bienvenue {{ name }} !",
+  "manager_hub_dashboard_feedback": "Nouveau dashboard : n'hésitez pas à nous <a href=\"{{ url }}\" target=\"_blank\">partager vos feedbacks</a>",
```

Comment:
```suggestion
  "manager_hub_dashboard_feedback": "Nouveau dashboard : n'hésitez pas à nous <a href=\"{{ url }}\" target=\"_blank\" rel=\"noopener\">partager vos feedbacks</a>",
```

Reply:
Important as it's an actual external link